libs/api/domains/driving-license/src/lib/graphql/models/qualityPhoto.model.ts

```diff
@@ -1,10 +1,13 @@
-import { Field, ObjectType } from '@nestjs/graphql'
+import { Field, ObjectType, ID } from '@nestjs/graphql'
 
 @ObjectType()
 export class QualityPhoto {
   @Field()
-  success!: boolean
+  hasQualityPhoto!: boolean
 
   @Field({ nullable: true })
-  qualityPhoto!: string | null
+  qualityPhotoDataUri?: string
```

Comment:
Just `dataUri`?
Since the GraphQL type is named QualityPhoto, the fields don't have to be prefixed with the same name.